### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by valvgab-bh**
Improve documentation

### CODE REVIEW

**Comment by valvgab-bh** [comment]
@delchiaro-bh please have a look at the file headers, modify if needed

File: src/cocohelper/adapters/dataset_adapter.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"""A generic adapter for converting datasets an arbitrary format to COCO format.
```
Comment (delchiaro-bh) [inline src/cocohelper/adapters/dataset_adapter.py:1]: The abstraction of an Adapter ...

File: src/cocohelper/__init__.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+"""A Python API to make working with COCO simpler.
```
Comment (delchiaro-bh) [inline src/cocohelper/__init__.py:1]: I would change to: with the COCO dataset format

File: src/cocohelper/errors/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"""Custom errors for the library.
```
Comment (delchiaro-bh) [inline src/cocohelper/errors/__init__.py:1]: I suggest to use *exception* word for documenting, it's easier to understand for everybody

File: src/cocohelper/errors/not_found_error.py [hunk 4]
```diff
@@ -1,3 +1,6 @@
+"""Errors for not found items in the COCO dataset.
```
Comment (delchiaro-bh) [inline src/cocohelper/errors/not_found_error.py:1]: Exceptions raised when items are not found in COCO dataset

File: src/cocohelper/errors/validation_error.py [hunk 5]
```diff
@@ -1,3 +1,6 @@
+"""Errors for non-valid COCO dataset.
```
Comment (delchiaro-bh) [inline src/cocohelper/errors/validation_error.py:1]: Exception raised when assertion of dataset validity fails

File: src/cocohelper/filters/cocofilters/__init__.py [hunk 6]
```diff
@@ -1 +1,3 @@
+"""Filters for the COCO data.
```
Comment (delchiaro-bh) [inline src/cocohelper/filters/cocofilters/__init__.py:1]: Special and complex filters to obtain specific data from COCO tables using the COCOHelper interface.

File: src/cocohelper/filters/cocofilters/cocofilters.py [hunk 7]
```diff
@@ -1,3 +1,5 @@
+"""Filters for the COCO images, annotations, and categories.
```
Comment (delchiaro-bh) [inline src/cocohelper/filters/cocofilters/cocofilters.py:1]: Special and complex filters used to obtain specific data from COCO `images`, `annotations` and `categories` tables.

File: src/cocohelper/filters/strategies/__init__.py [hunk 8]
```diff
@@ -1,2 +1,4 @@
+"""Filtering strategies.
```
Comment (delchiaro-bh) [inline src/cocohelper/filters/strategies/__init__.py:1]: Filtering strategies for ValueFilter and RangeFilter.

File: src/cocohelper/utils/path.py [hunk 9]
```diff
@@ -1,3 +1,5 @@
+"""Utils paths.
```
Comment (delchiaro-bh) [inline src/cocohelper/utils/path.py:1]: Utilities* for path manipulation.

File: src/cocohelper/utils/segmentation.py [hunk 10]
```diff
@@ -1,3 +1,5 @@
+"""Utils for converting segmentation annotations between different formats.
```
Comment (delchiaro-bh) [inline src/cocohelper/utils/segmentation.py:1]: Utilities*

**Review by delchiaro-bh — CHANGES_REQUESTED** [review]
Some of the docstrings are not so helpful to the library user, I made some comments.

If the docstring is a single line, please close the docstring in the same line:
```
"""Example of single-line docstring."""
```

Otherwise start from a new line and close in a new line:
```
"""
Example of multi-line docstring.
"""
```

**Comment by valvgab-bh** [comment]
@delchiaro-bh everything should be fixed now

**Pull request merged**